soes/esc.c

```diff
@@ -546,7 +536,7 @@ uint8_t ESC_mbxprocess (void)
    }
 
    /* SM0/1 access or SMn change event */
-   if (ESCvar.ALevent & ESCREG_ALEVENT_SM_MASK)
```

Comment:
ESCREG_ALEVENT_SM_MASK is no longer used anywhere, remove?

Reply:
Could still be used in user application. Be very careful with removing constants.

Reply:
Remove "or SMn change event" in comment

Reply:
Shouldn't the purpose of SOES be to abstract away implementation details like register contents from the user? It's no big deal though, leave it if it will be useful.